phoenicis-repository/src/main/java/org/phoenicis/repository/repositoryTypes/LocalRepository.java

```diff
@@ -239,6 +279,15 @@ public RepositoryDTO fetchInstallableApplications() {
         return results;
     }
 
+    private TypeDTO unSerializeType(File jsonFile) {
```

Comment:
Yes. But we should change that separately (I used the same name as in `CategoryDTO`).